src/hotspot/share/gc/g1/g1FullCollector.hpp

```diff
@@ -95,11 +98,13 @@ class G1FullCollector : StackObj {
   G1FullGCCompactionPoint* serial_compaction_point() { return &_serial_compaction_point; }
   G1CMBitMap*              mark_bitmap();
   ReferenceProcessor*      reference_processor();
+  size_t                   live_words(uint region_index) { return _live_stats[region_index]._live_words; }
```

Comment:
Could you add range check assertion for `region_index`? The extra spaces can be removed; no need to align with previous methods.